lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -27,7 +27,7 @@ class CacheHeaders extends Audit {
       failureDescription: 'Uses inefficient cache policy on static assets',
       helpText:
         'A long cache lifetime can speed up repeat visits to your page. ' +
-        '[Learn more](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#cache-control).',
+        '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/cache-policy).',
```

Comment:
we might want to update this one, we generally want folks to cache for much longer when possible

1 year for immutable static assets, i.e. configure webpack/gulp to embed a hash in the filename so they're unique (ideal)
explicitly add `no-cache` if the resource changes and freshness matters

Reply:
FYI https://github.com/google/WebFundamentals/issues/6285 is tracking the work to address this feedback in the doc